modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/1_application.json

```diff
@@ -305,6 +305,466 @@
+			"email": "chambribe1c@macromedia.com",
+			"externalReferenceCode": "RAYAPP-58",
+			"firstName": "Cleon",
+			"lastName": "Hambribe",
+			"phone": "650-182-8916",
+			"productCategory": "",
+			"productId": "RAYAP-001",
+			"productName": "Auto",
+			"r_accountToApplications_accountEntryERC": "RAYACCOUNT-019",
+			"state": "CA",
+			"submitDate": "2022-05-21",
+			"user": "",
+			"website": "Null",
+			"zip": "94064"
+		},
+		{
+			"address": "25709 Artisan Point",
+			"addressApt": "20",
+			"applicationCreateDate": "2022-10-30",
+			"applicationStatus": "bound",
+			"businessCategoryId": "",
+			"city": "Buffalo",
+			"dataJSON": "{\"driverInfo\":{\"form\":{\"ageFirstLicensed\":21,\"hasAccidentOrCitations\":\"no\",\"accidentCitation\":null,\"firstName\":\"Jessa\",\"gender\":\"Female\",\"lastName\":\"Capron\",\"maritalStatus\":\"Single\",\"millitaryAffiliation\":null,\"occupation\":\"Accountant\",\"governmentAffliation\":\"ABC Sherriff Association\",\"otherOccupation\":null,\"relationToContact\":\"Self\",\"highestEducation\":\"College\",\"creditRating\":\"Good\"}},\"vehicleInfo\":{\"form\":{\"annualMileage\":48893,\"year\":2017,\"make\":\"Toyota\",\"model\":\"Camry Hybrid XLE\",\"features\":\"Anti-lock braking system that is factory-installed,Lane-Departure Warning,Blind-Spot Monitoring\",\"ownership\":\"own\",\"primaryusage\":\"pleasure\"}},\"coverage\":{\"form\":{\"bodilyInjury\":\"$30,000 / $60,000\",\"medical\":\"$5,000\",\"propertyDamage\":\"$25,000\",\"uninsuredOrUnderinsuredMBI\":\"$25,000\",\"uninsuredOrUnderinsuredMPD\":\"$30,000 / $60,000\"},\"vehicles\":{\"collision\":\"$2,000\",\"comprehensive\":\"$750\"}}}",
+			"email": "jcapronb@cisco.com",
+			"externalReferenceCode": "RAYAPP-59",
+			"firstName": "Jessa",
+			"lastName": "Capron",
+			"phone": "716-189-3307",
+			"productCategory": "",
+			"productId": "RAYAP-001",
+			"productName": "Auto",
+			"r_accountToApplications_accountEntryERC": "RAYACCOUNT-020",
+			"state": "NY",
+			"submitDate": "2022-10-30",
+			"user": "",
+			"website": "Null",
```

Comment:
Why do you use "null" as a string value? Do you need this information or can you remove this property?

Reply:
It came from JSON file provided by Matt, I think it can be removed without effect the result.